src/SleighAsm.cpp

```diff
@@ -322,7 +474,13 @@ int SleighAsm::disassemble(RAsmOp *op, unsigned long long offset)
 	try
 	{
 		length = trans.printAssembly(assem, addr);
-		r_strbuf_set(&op->buf_asm, assem.str);
+		// r_strbuf_set(&op->buf_asm, assem.str);
+		auto *ins = trans.getInstruction(addr);
+		stringstream ss;
+		ss << assem.str << " " << ins->printFlowType(ins->getFlowType());
```

Comment:
Is this debugging code? If you want to keep it, you can wrap it in `#if 0` to disable

Reply:
Yes, will do.